cmd/server/main.go

```diff
@@ -278,7 +324,8 @@ func handleAddSecret(obj runtime.Object) {
 			clientId := string(secret.Data[BindingClientId])
 			cert := string(secret.Data[IOSCert])
 			passPhrase := string(secret.Data[IOSPassPhrase])
-			handleIOSVariant(clientId, cert, passPhrase)
+			variantReferenceId := string(secret.Data[VariantReferenceId])
+			handleIOSVariant(clientId, cert, passPhrase, variantReferenceId)
```

Comment:
Instead of passing four params, maybe we should just pass a pointer to the secret?